### DISCUSSION TITLE
Custom domain Github pages

### DISCUSSION THREAD

**Discussion by techlxrd** [community/community#29175] [New to GitHub]
Hello! I need some help, I'm trying to add a custom domain from freenom and it's stuck on DNS Check in Progress

**Comment by queenofcorgis** [comment]
Hi there @Outlander2007 👋🏼 

Are you still experiencing this?

**Comment by luckylittle** [comment]
@Outlander2007 DNS changes can take up to 24 hours to propagate. Have you [Verified your custom domain for GitHub Pages](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/verifying-your-custom-domain-for-github-pages)?

Can you give us your domain so we can check `dig WWW.EXAMPLE.COM +nostats +nocomments +nocmd`?

Be aware there are custom domain names that are unsupported. Make sure your site does not:
* Use more than one apex domain. For example, both `example.com` and `anotherexample.com`.
* Use more than one `www` subdomain. For example, both `www.example.com` and `www.anotherexample.com`.
* Use both an apex domain and custom subdomain. For example, both `example.com` and `docs.example.com`.
(The one exception is the `www` subdomain. If configured correctly, the `www` subdomain is automatically redirected to the apex domain.)

**Reply by techlxrd** [reply]
Yeah, I looked in there but when I add records two of them give error

**Reply by techlxrd** [reply]
My domain is silic0nhub.ga

**Reply by iuriimattos** [reply]
@Outland3r2007 you issue as solved? have same issue my domain is http://iurimatos.com.br/

**Comment by techlxrd** [comment]
I solved it

Sent from my iPhone

On 30 Jan 2023, at 22:07, Iuri Matos ***@***.***> wrote:

﻿

@Outland3r2007<https://github.com/Outland3r2007> you issue as solved? have same issue my domain is http://iurimatos.com.br/

**Comment by CandidDeer** [comment]
I have the same problem - may I add to this thread? My domain is https://ltdesign.work/ and the DNS check has been stuck on "in progress" for around 2 weeks now... 😞 Would love if any of you could help me? Thx!

**Reply by techlxrd** [reply]
Your website works

**Reply by CandidDeer** [reply]
Yes, I know this link does, but the GitHub DNS check status is still stuck on "in progress" and therefore I cannot enforce https. So when someone goes to [www.ltdesign.work](url), they are shown "Connection not secure". Therefore I need to enforce the https through GitHub, which I cannot do as long as DNS check is not complete.

**Reply by CandidDeer** [reply]
Ah, I think I fixed the issue on the Cloudflare end - they offer https enforcement too.
Still wondering why my GitHub DNS check status is stuck on "in progress".

**Comment by Cristina-Assuncao** [comment]
I am stuck with the same problem. The GitHub DNS check status has been stuck on "in progress". My domain is www.cristina-assuncao.com
Can anyone advise please?

**Reply by CandidDeer** [reply]
@Cristina-Assuncao Mine actually never did move from "in progress". No idea why.

**Reply by Cristina-Assuncao** [reply]
I see, but my website doesn't work! In Firefox I get the following error:

"Hmm. We’re having trouble finding that site.
We can’t connect to the server at www.cristina-assuncao.com.
If you entered the right address, you can:
    Try again later
    Check your network connection
    Check that Firefox has permission to access the web (you might be connected but behind a firewall)"

Other error messages appear in different browsers.

I can't figure out why.

**Comment by Madhav-Malhotra** [comment]
Here's what solved the issue for me: 

Originally, I put `www.example.com` as my custom domain. I'd waited a day, but Github just showed 'DNS Check in Progress'. Then, I clicked 'Remove', entered `example.com` (no www subdomain), and clicked Save. The DNS check finished in 25 seconds for me. 

Here are screenshots of my Github pages settings and my DNS configuration:
![My Github Pages Settings](https://i.ibb.co/ckcQ9WV/image.png)
![My DNS Settings](https://i.ibb.co/tPHKcPc/image.png)
